src/renderer/components/PathInput.tsx

```diff
@@ -30,9 +32,13 @@ export const PathInput = React.forwardRef<HTMLInputElement, PathInputProps>((pro
         <CustomInput inputRef={ref} disabled={true} value={value} placeholder={placeholder} />
         {endAdornment}
       </InputContainer>
-      <Button color="secondary" variant="contained" onClick={onClick} disabled={disabled}>
-        Select
-      </Button>
+      <Tooltip title={(disabled && tooltipText) ?? ""}>
```

Comment:
Why is this necessary to check for disabled? Why can't we just pass the tooltip text straight in?

```suggestion
      <Tooltip title={tooltipText}>
```

Reply:
right that tooltip is kinda specific for the disabled case. I wanted to move it outside the component but that didn't work cause it would be on the entire component. We could just show the tooltip at all times I guess.

Reply:
Couldn't the component passing in the `tooltipText` to `PathInput` pass in undefined if not `disabled` then?

Reply:
can't be undefined, but yeah it can pass in an empty string